src/actions/assignment.js

```diff
@@ -69,3 +89,37 @@ export function createAssignment(assignment, classroomId) {
     .catch(response => console.log('RESPONSE-error: ', response));
   };
 }
+
+export function fetchAssignments() {
+  console.log('-'.repeat(40), '\nfetchAssignments()\n');
+  return dispatch => {
+    dispatch({
+      type: types.REQUEST_ASSIGNMENTS,
+    });
+    return fetch(root + assignments, {
+      method: 'GET',
+      mode: 'cors',
+      headers: new Headers({
+          'Authorization': apiClient.headers.Authorization,
+          'Content-Type': 'application/json'
+        })
+      })
+      .then(response => response.json())
+      .then(json => {
+        console.log('!'.repeat(80), '\n', json.data);
+        dispatch({
```

Comment:
Ah, so most of the logic should be inside Reducers, not Actions? I'm game to change this, but the Assignments are based on the template set by Classrooms, so this may need to be part of a larger code cleanup